src/components/modebar/buttons.js

```diff
@@ -618,6 +618,42 @@ modeBarButtons.resetViewMapbox = {
     }
 };
 
+modeBarButtons.zoomInMapbox = {
+    name: 'zoomInMapbox',
+    title: function(gd) { return _(gd, 'Zoom in'); },
+    attr: 'zoom',
+    val: 'in',
+    icon: Icons.zoom_plus,
+    click: handleMapboxZoom
+};
+
+modeBarButtons.zoomOutMapbox = {
+    name: 'zoomOutMapbox',
+    title: function(gd) { return _(gd, 'Zoom out'); },
+    attr: 'zoom',
+    val: 'out',
+    icon: Icons.zoom_minus,
+    click: handleMapboxZoom
+};
+
+function handleMapboxZoom(gd, ev) {
+    var button = ev.currentTarget;
+    var val = button.getAttribute('data-val');
+    var fullLayout = gd._fullLayout;
+    var subplotIds = fullLayout._subplots.mapbox || [];
+    var scalar = 1.3;
```

Comment:
This is changing fast (specially when having angles for display urban areas).
I suggest using `1.02` instead of `1.3` here.

Reply:
Right, but I thought value lower than 1.3 were "too slow" for world-scope maps.

We could introduce some sort of non-linear scale, but then that might be too inconsistent with the other zoom in/out buttons we have.

Reply:
I don't think a constant close to `1.02` is "too slow" for world maps.
Would you like me to make a codepen to help compare the two?

Reply:
> I suggest using `1.02`

> Would you like me to make a codepen to help compare the two?

I'm curious, how did you come to suggesting `1.02` with testing it beforehand?

Reply:
I tested it by changing it on the local branch.

Reply:
Ok, so sounds like you should be the one making a codepen

Reply:
My pleasure :slightly_smiling_face:

Reply:
[Demo](https://codepen.io/MojtabaSamimi/pen/VwYwxPo?editors=1000) to compare zoom speed 1.3 on the left with 1.02 on the right.

Reply:
@archmoj I picked `1.05` in my latest commit - which is somewhere between our two values.

Try out: https://codepen.io/etpinard/pen/wvBWpRQ

Is that good enough for you?